Bump wireguard-manager from 06-20-2021 to 10-26-2021

Prajwal-Koirala opened the pull request:
# Description

- More distro support
- added an optional content blocker that blocks ads and trackers and other stuff.
- fixed all the dependency issues.
- fixed the issue with unbound and ipv6
- now the server can have as many clients as they want.
- fixed the uninstall
- also the thing that people hated where it would remove things without asking, that has been removed.
- a lot of other bugs.

EvilOlaf commented:
Info: https://github.com/complexorganizations/wireguard-manager/releases/tag/v1.0.0.10-26-2021

Prajwal-Koirala commented:
> Info: https://github.com/complexorganizations/wireguard-manager/releases/tag/v1.0.0.10-26-2021

Notes:
- More distro support
- added an optional content blocker that blocks ads and trackers and other stuff.
- fixed all the dependency issues.
- fixed the issue with unbound and ipv6
- now the server can have as many clients as they want.
- fixed the uninstall
- also the thing that people hated where it would remove things without asking, that has been removed.
- alot of other bugs.
